fiftyone/server/metadata.py

```diff
@@ -53,10 +53,21 @@ async def get_metadata(
         metadata dict
     """
     filepath = sample["filepath"]
-    is_video = media_type == fom.VIDEO
     metadata = sample.get("metadata", None)
     urls = _create_media_urls(collection, sample, url_cache)
 
+    has_orthographic_projection_metadata = (
```

Comment:
Here you're handling the case where the collection's `media_type` is `point-cloud`, but we could also now handle the case where the collection's `media_type` is `group` but the user has selected a point cloud slice to visualize in the grid via the group slice selector.

Currently, the group slice selector only includes image/video slices, but that logic could be implemented to also include any point cloud slices if the dataset has an `OrthographicProjectionMetadata` field.

This can be a separate PR if appropriate.